### ISSUE TITLE
"Where are the modules?" - Please read

### ISSUE THREAD

**Issue by nathanjhood** [StoneyDSP/StoneyVCV#8]
# Bug report

Hi! 

If you are reading this - thanks! You might have even tried to build my VCV Rack plugin (successfully, I hope) and found that no plugin or modules show up in your VCV installation. Why is that?

It's because I'm currently working on a testing/deployment framework using GitHub Actions to build the plugin for all target platforms (currently: win-x64, osx-x64, and lin-x64) while consuming my own DSP library as a dependency. My first task in this is to make sure the DSP library compiles in the recommended Rack environment for each platform. Once the DSP and other deps are building an empty plugin successfully (we're basically there), I'll begin adding modules on a `feat/<moduleName>` branch each. 

First up, you'll be seeing some functional but probably quite ugly modules within the next few days, as I'm focused on the internals - particularly, the DSP - first and foremost, and I'm working alone, in my free time :)

Please feel welcome to check my `preview` (beta) and `development` (experimental) branches at any time.

StoneyVCV shall likely remain at v2.0.0 until I feel ready to submit the first modules to the VCV library. Please keep this in mind if re-visiting and trying a new build of StoneyVCV (the version number acts as a logic switch on module availability).

Thanks and stay tuned!

Code of Conduct
By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/StoneyDSP/.github/blob/production/CODE_OF_CONDUCT.md).

**Comment by nathanjhood** [comment]
Current status:

I'm a little blocked by #21 

tl:dr; I kind of *insist* on writing tests for anything that passes, or touches, the audio stream; parameters and their props such as scaling, min/max values, etc; as well for anything that uses SIMD, and would ideally aim for 100% test coverage of any *customary* code; I'm not interested in testing VCV Rack itself, that's their job - ideally, any inherited functions which we specify an *override* for should be on the testing list, along with whatever whacky ideas of our own we end up implementing in our `Module`/`ModuleWidget`/`Widget` classes.

Disclaimer: Upskilling on unit-testing audio software written in C++ was/is my actual main motivation in this project, above anything else about it. I'm basically interested in writing modules for having something to write tests for.

There's something about those `.dll.a`'s on MinGW... I don't know what I don't know, and I'm losing time/patience in not figuring it out yet. I'll probably reach out for help at the forums; meanwhile, I will probably add some module code *anyway* - tests be damned - on an offshoot branch from `development`.

There's a whole component library to be done, and I'd like to keep my own enthusiasm up while I figure out how to finish the vcpkg/Catch2 branch asynchronously.

So, stay tuned for module code; it may not be on the default branch until later, though.

**Comment by nathanjhood** [comment]
With regards to:

> There's a whole component library to be done, and I'd like to keep my own enthusiasm up while I figure out how to finish the vcpkg/Catch2 branch asynchronously.
> 
> So, stay tuned for module code; it may not be on the default branch until later, though.

I'll be starting with... :drum: some blank panels!

Oh I know... after all that... slow start, eh? But I only want some basic idea of the dimensions to use and how I should structure my frame buffer widgets and `draw()` methods... also wanna work out how to make custom `SVGScrew`s and such forth, which looks easy and interesting in a "curious-ly recurring" way.

Having a good idea of the GUI real estate available will inform my decisions about many things; params, in/out ports... I'm not too interested in scaling up a whole design system right off the bat, though. Let's go!

**Comment by nathanjhood** [comment]
Ok, blockers have been dealt with. I've got a short break planned now, the modules will be hitting the unit tests in a couple more days.
